### PR TITLE
issue#2 added a CI pipeline

### PR DESCRIPTION
**Pull request by timberhill**

### CODE REVIEW

**Comment by timberhill** [comment]
@danyloid should be ready now lol

File: README.md [hunk 1]
```diff
@@ -31,5 +31,5 @@ The code sends a POST request to endpoints in `config.json` with
 JSON payload, same for both comments and posts, e.g.:
 
 ```json
-{'id': 'j643al2', 'created_utc': 1674835189.0}
+{"id": "j643al2", "created_utc": 1674835189.0}
```
Comment (danyloid) [inline README.md:34]: @timberhill just noticed this is not an ISO string :)

**Review by danyloid — APPROVED** [review]
LGTM

**Comment by timberhill** [comment]
Image version pushed: https://github.com/flam-flam/dispatcher-service/pkgs/container/dispatcher/67629464?tag=0.1.0
Repo tagged: https://github.com/flam-flam/dispatcher-service/releases/tag/0.1.0

**Pull request merged**